src/contracts/pods/EigenPod.sol

```diff
@@ -437,20 +437,24 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         IEigenPod.VerifiedPartialWithdrawal calldata verifiedPartialWithdrawal,
         uint64 feeGwei,
         address feeRecipient
-    ) external onlyEigenPodManager onlyWhenNotPaused(PAUSED_EIGENPODS_VERIFY_WITHDRAWAL) {
+    ) external onlyEigenPodManager onlyWhenNotPaused(PAUSED_EIGENPODS_FULFILL_PARTIAL_WITHDRAWAL_PROOF_REQUEST) {
 
         require(verifiedPartialWithdrawal.mostRecentWithdrawalTimestamp == mostRecentWithdrawalTimestamp, "EigenPod.fulfillPartialWithdrawalProofRequest: proven mostRecentWithdrawalTimestamp must match mostRecentWithdrawalTimestamp in the EigenPod");
         require(mostRecentWithdrawalTimestamp < verifiedPartialWithdrawal.endTimestamp, "EigenPod.fulfillPartialWithdrawalProofRequest: mostRecentWithdrawalTimestamp must precede endTimestamp");
 
-        require(sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei + feeGwei <= verifiedPartialWithdrawal.provenPartialWithdrawalSumGwei, "EigenPod.fulfillPartialWithdrawalProofRequest: proven sum must be less than or equal to provenPartialWithdrawalSumGwei + feeGwei");
+        require(sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei <= verifiedPartialWithdrawal.provenPartialWithdrawalSumGwei - feeGwei, "EigenPod.fulfillPartialWithdrawalProofRequest: proven sum must be less than or equal to provenPartialWithdrawalSumGwei + feeGwei");
         uint64 provenPartialWithdrawalSumGwei = verifiedPartialWithdrawal.provenPartialWithdrawalSumGwei;
         provenPartialWithdrawalSumGwei -= feeGwei;
 
 
         // subtract an partial withdrawals that may have been claimed via merkle proofs
-        provenPartialWithdrawalSumGwei -= sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei;
-        sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei = 0;
-        _sendETH_AsDelayedWithdrawal(podOwner, provenPartialWithdrawalSumGwei);
+        if(provenPartialWithdrawalSumGwei > sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei){
+            provenPartialWithdrawalSumGwei -= sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei;
```

Comment:
again, a lot of this is conditional on `sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei == 0`, we should condition on that IMO to save gas